pinot-controller/src/test/java/org/apache/pinot/controller/helix/PinotControllerModeTest.java

```diff
@@ -56,7 +56,7 @@ public void testHelixOnlyController() {
     stopController();
   }
 
-  @Test
+  @Test(enabled = false)
```

Comment:
Don't disable this test, we need to fix it

Reply:
The test will be refactored in this PR:https://github.com/apache/incubator-pinot/pull/4323
We don't need to enable/disable resource from ideal state any more.
We'll re-enable the test in that PR.